book/tut2_spec_pd_gpd.ipynb

```diff
@@ -0,0 +1,961 @@
+      "text/plain": [
+       "<IPython.core.display.Image object>"
+      ]
+     },
+     "metadata": {
+      "image/png": {
+       "width": 500
+      }
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "fig = pygmt.Figure()\n",
+    "fig.basemap(region=[-88, -87.4, 41.6, 42.05], projection=\"M10c\", frame=\"rltb\")\n",
+    "\n",
+    "# Set up colormap\n",
+    "popul_min = np.min(gdf_airbnb[\"population\"])\n",
+    "popul_max = np.max(gdf_airbnb[\"population\"])\n",
+    "pygmt.makecpt(cmap=\"bilbao\", series=[popul_min, popul_max, 10])\n",
+    "# Add colorbar\n",
+    "fig.colorbar(frame=\"x+lpopulation\")\n",
+    "\n",
+    "# Plot the polygons with color-coding for the population\n",
+    "fig.plot(\n",
+    "    data=gdf_airbnb, \n",
+    "    pen=\"0.2p,gray10\", \n",
+    "    fill=\"+z\", \n",
+    "    cmap=True,\n",
+    "    aspatial=\"Z=population\",\n",
+    ")\n",
+    "\n",
+    "fig.show(dpi=150)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "03516fe8-5013-4acb-89d1-a1ebdbc1d426",
+   "metadata": {},
+   "source": [
+    "### 3. Additional comments\n",
+    "\n",
+    "Some interesting aspects:\n",
+    "\n",
+    "- Convert other objects to `pandas` or `GeoPandas` objects to make them usable in PyGMT\n",
+    "- Combination with `shapely` to create more complex geometries (i.e. `from shapely.geometry import Polygon`)\n",
+    "- Use sutiable colormaps (F. Crameri publication)\n",
```

Comment:
If you want, it is possible to add a citation to the `book/references.bib` file, see example usage at https://github.com/GenericMappingTools/agu24workshop/blob/3b4bb5016037f9f613403a50aa7bcc5e1e8ed210/book/markdown.md#citations

```suggestion
    "- Use suitable Scientific colormap (F. Crameri publication)\n",
```

Reply:
Updated in commit https://github.com/GenericMappingTools/agu24workshop/pull/4/commits/f5d8a0d42bbcc633d9c557c4942cd5ca436026a9 and also added the related publications.